str/gangstr_runner.py

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env python
+
+"""
+Example of running Batch script with analysis-runner.
+"""
+import os
+import hailtop.batch as hb
+import click
+
+DATASET = os.getenv('DATASET')
+HAIL_BUCKET = os.getenv('HAIL_BUCKET')
+OUTPUT_SUFFIX = os.getenv('OUTPUT')
+BILLING_PROJECT = os.getenv('HAIL_BILLING_PROJECT')
+ACCESS_LEVEL = os.getenv('ACCESS_LEVEL')
+
+REF_FASTA = 'gs://cpg-reference/hg38/v1/Homo_sapiens_assembly38.fasta'
+SAMTOOLS_IMAGE = 'australia-southeast1-docker.pkg.dev/cpg-common/images/samtools:v0'
+GANGSTR_IMAGE = "australia-southeast1-docker.pkg.dev/cpg-common/images/gangstr:v2.5"
+regions = 'gs://cpg-fewgenomes-test/hoptan-str/gangstr_catalog_with_offtarget.bed'
+
+
+@click.command()
+@click.argument('cram_path')
+@click.argument('region')
+def main(cram_path: str, region: str):  # pylint: disable=missing-function-docstring
+    """
+    Subset CRAM or BAM file CRAM_PATH to REGION. Example: batch.py sample.cram chr21:1-10000
+    """
+    # Initializing Batch
+    backend = hb.ServiceBackend(billing_project=BILLING_PROJECT, bucket=HAIL_BUCKET)
+    b = hb.Batch(backend=backend, default_image=os.getenv('DRIVER_IMAGE'))
+
+    # Adding a job and giving it a descriptive name.
+    j = b.new_job('Subset CRAM')
+
+    # Making sure Hail Batch would localize both CRAM and the correponding CRAI index
+    cram = b.read_input_group(**{'cram': cram_path, 'cram.crai': cram_path + '.crai'})
+
```

Comment:
As @MattWellie mentioned, you'll want to load the region file in:

```suggestion
region = b.read_input(region_path)
```

Reply:
There are two 'region-like' variables. I should change their names so it's more clear. One refers to the variant catalog, which is a bed file. The other is the second command line argument specifying a subset of the chromosome region if needed.